backend/scripts/loadEcNumbers.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Downloads the enzymes.dat file and loads them into the ec_numbers table
+
+mkdir ../../../data/ecnumbers
+datadir="../../../data/ecnumbers"
+currentdir=`pwd`
+
+# prepare the sql file
+cd "${datadir}"
+rm -f enzyme.dat
+rm -f enzclass.txt
+rm -f enzyme.sql
+wget ftp://ftp.expasy.org/databases/enzyme/enzyme.dat
+wget ftp://ftp.expasy.org/databases/enzyme/enzclass.txt
+
+echo "TRUNCATE TABLE ec_numbers;" > enzyme.sql
+awk 'BEGIN {
+  FS = "   "
+}
+/^ID/{
+  if (id != "") {
+    printf("INSERT INTO ec_numbers (number, name) VALUES (\"%s\", \"%s\");\n", id, name)
+  }
+  name = ""
+  id = $2
+}
+/^DE/{
+  gsub(/.$/, "", $2)
+  name = name $2
+}
+END {
+  printf("INSERT INTO ec_numbers (number, name) VALUES (\"%s\", \"%s\");\n", id, name)
+}' enzyme.dat >> enzyme.sql
+
+cat enzclass.txt| sed -e "s/- /-    /" | awk 'BEGIN {
+        FS = "    "
+}
+/^[1-9]/ {
+
```

Comment:
remove newline


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3370) by @bmesuere on Mon Oct 19 2015 at 14:55._